nextflow.config

```
@@ -16,6 +16,7 @@ params {
     // short read preprocessing options
     skip_clipping                        = false
     clip_tool                            = 'fastp'
+    clip_save_reads                      = false
```

All other related params start with "save" and that makes it easier to recognize, imho, but this one has a different naming pattern. I'd also change this throughout like
```suggestion
    save_clipped_reads                      = false
```